pyproject.toml

```diff
@@ -25,6 +25,7 @@ python = "^3.7"
 httpx = {version = ">=0.15.0 <1.0.0", optional = true}
 aiohttp = {version = "^3.6.2", optional = true}
 yarl = "^1.4.2"
+xmltodict = '^0.13.0'
```

Comment:
I'm not too sure about this dependency. xml is scary to touch and from looking at the implementation this doesn't use something like [defusedxml](https://github.com/tiran/defusedxml) which hardens xml parsers in Python. It looks like you're also only interested in a few values in the XML returned from AWS (and the XML itself looks rather trivial), so using defusedxml + one of the stdlib xml parsers should not be too difficult.

Reply:
I'm not a big fan of xml, so I use a library to avoid contact with it. I make an implementation that takes credentials from aws response. Comment please, I don't have enough practice with it.